v-next/hardhat-utils/src/ci.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Checks whether the current process is running in a CI environment.
+ *
+ * @returns True if the current process is running in a CI environment.
+ */
+export function isCi() {
+  const env = process.env;
+
+  return !!(
```

Comment:
`!!` of boolean doesn't add anything

Reply:
[Done in this commit](https://github.com/NomicFoundation/hardhat/pull/5340/commits/d5ce13f963d5f7450bb946cb3c0933e48f895730)